share/jupyter/voila/templates/classic/index.html.j2

```diff
@@ -32,6 +32,15 @@
 {%- block html_head_css -%}
   {{ resources.include_css("static/materialcolors.css") }}
   {{ resources.include_css("static/labvariables.css") }}
+
+  {#
+    This is a temporary solution, when Notebook 7 is out, the "classic"
+    template should inherit from the "lab" template and get the lab CSS
+    automatically
+  #}
+  {{ resources.include_css("../lab/static/index.css") }}
```

Comment:
Although this fixes the missing CSS for some mimetype renderning, it's breaking the global look of the classic template (and opinionated templates like the AdminLTE one and the ipyvuetify one would be broken too) .

I'll see if I can be more specific about the CSS that is included in the page, we should only include the CSS for the mimetype renderers.